components/back-top/index.tsx

```diff
@@ -40,10 +40,6 @@ export interface BackTopProps {
 }
 
 export default class BackTop extends React.Component<BackTopProps, any> {
-  static defaultProps = {
-    visibilityHeight: 400,
-  };
```

Comment:
I came across this problem too... 

https://github.com/ant-design/ant-design-mobile/commit/e5d1a708f16076a879fee5f49dbe63e9f6d5c90f#commitcomment-20970016

seems like Typescript type-inference will  not handle React defaultProps, and it's actually
reasonable， since  `defaultProps` do not belongs to any EcmaScript Standard